src/main/java/com/cloudbees/jenkins/support/impl/AboutJenkins.java

```diff
@@ -1010,4 +1004,24 @@ private String getLabelString(Node n) {
         }
     }
 
+    /**
+     * Fixes JENKINS-47779 caused by JENKINS-47713
+     * Not using SortedSet because of PluginWrapper doesn't implements equals and hashCode.
+     * @return new copy of the PluginManager.getPlugins sorted
+     */
+    private static Iterable<PluginWrapper> getPluginsSorted() {
+        PluginManager pluginManager = Helper.getActiveInstance().getPluginManager();
+        return getPluginsSorted(pluginManager);
+    }
+
+    private static Iterable<PluginWrapper> getPluginsSorted(PluginManager pluginManager) {
+        return listToSortedIterable(pluginManager.getPlugins());
+    }
+
+
+    private static <T extends Comparable<T>> Iterable<T> listToSortedIterable(List<T> list) {
+        final List<T> sorted = new LinkedList<T>(list);
+        Collections.sort(sorted);
```

Comment:
it's not the most effective way of sorting, but OK.

Reply:
It's the same way than before but suggestions are welcome :-)